experimental/web/sample_dynamic/main.c

```diff
@@ -49,8 +90,8 @@ void inspect_module(iree_vm_module_t* module) {
   for (iree_host_size_t i = 0; i < module_signature.export_function_count;
        ++i) {
     iree_vm_function_t function;
-    iree_status_t status = iree_vm_module_lookup_function_by_ordinal(
-        module, IREE_VM_FUNCTION_LINKAGE_EXPORT, i, &function);
+    IREE_IGNORE_ERROR(iree_vm_module_lookup_function_by_ordinal(
```

Comment:
don't ignore; you've got uninitialized memory in `function` if the lookup fails

Reply:
Thanks, I handled the error. This `inspect_module()` function is something I'd like to rework with JavaScript bindings. I think I want to populate a struct -> JSON object (or something like it) with all this metadata, to avoid needing to hop back and forth between JS and Wasm.